ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -130,14 +137,37 @@ private T GetSetting<T>(string key, T defaultValue)
                     return (T)data;
                 }
             }
+#else
+            if (typeof(T) == typeof(bool))
+            {
+                Boolean result = Boolean.TryParse(ConfigurationManager.AppSettings[key], out result);
```

Comment:
I have a PR for it already.

---
In reply to: [84324718](https://github.com/ReactWindows/react-native-windows/pull/809#discussion_r84324718) [](ancestors = 84324718)